src/components/App.jsx

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
```

Comment:
Do we still have a need for this file?

Reply:
This is going to be the component the landing page loads that keeps other child components